src/login.service.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  BadRequestException,
+  Injectable,
+  UnauthorizedException,
+} from '@nestjs/common';
+import { JwtService } from '@nestjs/jwt';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { User } from './entities/user.entity';
+import { JWT, JWTResult } from './typings/jwt.typing';
+
+@Injectable()
+export class LoginService {
+  constructor(
+    @InjectRepository(User)
+    private usersRepository: Repository<User>,
+    private jwtService: JwtService,
+  ) {}
```

Comment:
I suggest you sorting alphabetically:
```suggestion
  constructor(
    private jwtService: JwtService,
    @InjectRepository(User)
    private usersRepository: Repository<User>,
  ) {}
```

Reply:
Resolved on 3ece8dd3091cd40b9254d8fed7ac466ac23af613.